### PR TITLE
Import reactions from Slack

### LINKED ISSUE #54: Support importing reactions from Slack
At the moment, reactions to posts (via emojis) are not imported. It would be worth having them.

### PR DESCRIPTION
**Pull request by wetneb**
#### Summary

Adds support for importing reactions from Slack.

#### Ticket Link

Fixes #54.

### CODE REVIEW

**Review by fmartingr — APPROVED** [review]
👍  Thank you!

**Comment by wetneb** [comment]
I've added some small improvements after getting some errors during a Slack migration:
* the `+1::skin-tone-3` sorts of emojis are not supported by Mattermost, so we strip the `::skin-tone-3` part for the reaction to be imported successfully
* I got some validation error at import time ("Reaction CreateAt property must be greater than the parent post CreateAt") so I adjusted the import code slightly to add a small offset to the reaction times

File: services/slack/intermediate_test.go [hunk 1]
```diff
@@ -823,4 +823,93 @@ func TestTransformPosts(t *testing.T) {
...
+						Reactions: reactions2,
+					},
+				},
+			},
+		}
+
+		err := slackTransformer.TransformPosts(slackExport, "", false, false, false)
+		if err != nil {
+			t.Fatalf("expected no error, got %v", err)
+		}
```
Comment (isacikgoz) [inline services/slack/intermediate_test.go:872]:
```suggestion
		require.NoError(t, err)
```

**Review by isacikgoz — COMMENTED** [review]
@wetneb this looks good to me, thank you very much. Can you consider changing tests to `reuqire`? It's more convenient to read and generally that's the pattern we use.

**Pull request merged**